arches/app/views/api.py

```diff
@@ -110,6 +107,37 @@ def dispatch(self, request, *args, **kwargs):
         return super(APIBase, self).dispatch(request, *args, **kwargs)
 
 
+class GetFrontendI18NData(APIBase):
+    def get(self, request):
+        origin_entity_name = request.GET.get('originEntityName')
+
+        user_language = get_language()
+        if user_language is None:
+            user_language = "en"
```

Comment:
I think this can be omitted as `get_language()` will always return a valid language code

Reply:
fixed in latest